Cargo.toml

```diff
@@ -22,13 +22,13 @@ optional = true
 [features]
 std = []
 
-#[dev-dependencies]
-#criterion = "0.3"
-#
-#[[bench]]
-#name = "from_iterator"
-#harness = false
-#path = "benches/from_iterator.rs"
+[dev-dependencies]
+criterion = "0.3"
+
+[[bench]]
+name = "from_iterator"
+harness = false
+path = "benches/from_iterator.rs"
```

Comment:
The benchmark is commented out in Cargo.toml on purpose, at it increases compile times (on CI and fresh local builds, but not when used as a dep). Did these comments really break the build for you?!

Reply:
Yes, for some reason it tries to build also `from_iterator.rs`, and I did not think it might be commented out on purpose.
But it can be some weird IDE/OS thing, just tried from console and it does not run it.